locust/test/test_util.py

```diff
@@ -30,3 +30,8 @@ def test_rounding_up(self):
         self.assertEqual(4, proper_round(3.5))
         self.assertEqual(5, proper_round(4.5))
         self.assertEqual(6, proper_round(5.5))
+
+    def test_normalize_decimal(self):
+        self.assertEqual(1, normalize_decimal(1))
+        self.assertEqual(0.99, normalize_decimal(0.99))
+        self.assertEqual("a", normalize_decimal("a"))
```

Comment:
What is this function meant to do? From these tests it seems like the function is simply returning the inputted value :)

Reply:
But I'm of the opinion that we don't need to continue supporting the legacy UI, so then maybe this function isn't needed either?

Reply:
@andrewbaldwin44 
I agree that it's probably best not to continue adding new implementations to the legacy UI. 
Regarding the features we've already implemented in the legacy UI, should I remove them?

Reply:
In my opinion, I would prefer to remove them. As we are right now, we can consider the legacy UI stable. If we continue to add new features it always has the possibility of introducing bugs. In the interest of not having to support the old UI anymore, I would prefer to leave the code as it is. Let me know if that's ok with you :)

Reply:
OK, I've removed the new features from the legacy UI.
[0cb9a62](https://github.com/locustio/locust/pull/2550/commits/0cb9a62e3beebbf46c8218f593a6c7779bdfa29f)